django/conf/locale/de_CH/formats.py

```diff
@@ -27,7 +27,7 @@
 
 # these are the separators for non-monetary numbers. For monetary numbers,
 # the DECIMAL_SEPARATOR is a . (decimal point) and the THOUSAND_SEPARATOR is a
-# ' (single quote).
+# , (comma).
```

Comment:
I don't think it is a `comma` but also a `nbsp` as for plain numbers. (see my comment in the ticket). So I would think it would be correct to remove " and the THOUSAND_SEPARATOR is a x (whatever)." from the comment.

Reply:
I have added a new commit taking into account the latest changes in the thread